hibernate-core/src/main/java/org/hibernate/persister/collection/AbstractCollectionPersister.java

```diff
@@ -290,6 +293,7 @@ public AbstractCollectionPersister(
 		}
 		batchSize = batch;
 
+		this.configuredBatchSize = ConfigurationHelper.getInt(Environment.STATEMENT_BATCH_SIZE, this.factory.getProperties(),1);
```

Comment:
Did you take into consideration that we can override that on a per-Session basis now? Check `SharedSessionContract#setJdbcBatchSize`

Reply:
No i don't, I was searching for a better API from where i can get current configured jdbc batchsize, but i couldnt find it.